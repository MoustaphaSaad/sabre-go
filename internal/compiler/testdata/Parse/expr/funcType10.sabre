func(n int) func(x int)