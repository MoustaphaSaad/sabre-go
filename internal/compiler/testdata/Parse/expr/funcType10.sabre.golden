(FuncType
  (FuncType-Parameters
    (IdentifierExpr IDENTIFIER(n))
    (NamedType IDENTIFIER(int))
  )
  (FuncType-Results
    (FuncType
      (FuncType-Parameters
        (IdentifierExpr IDENTIFIER(x))
        (NamedType IDENTIFIER(int))
      )
    )
  )
)